content/contributing-code/usability/contents.lr

```diff
@@ -0,0 +1,60 @@
+_model: page
+---
+_template: page-with-toc.html
+---
+title: Usability
+---
+body:
+
+At CC, our goal is to make our products as easy and delightful to use as possible. To that end, we are constantly talking to our users. Sometimes our sessions involve understanding current experiences and identifying pain points, other times we’re determining upcoming feature prioritization. If we plan to roll out a new product, or design a new interface or feature for an existing product, we do research! We appreciate our users volunteering their time to help us with this research by participating in test sessions.
+
+## Participate in a Session
+When we’re rolling out a specific feature, we do **usability tests** to test the proposed experience.
+
+Keep an eye on the `#cc-usability` channel in the [Creative Commons Slack](https://slack-signup.creativecommons.org/) or sign up for the [CC Usability Announcements list](https://groups.google.com/a/creativecommons.org/forum/#!forum/usability-announce). That way you’ll be the first to hear when there’s a round of usability tests being scheduled for something new.
+
+At any point in time, we’re engaging with our users through **user interviews**, where we learn more about attitudes towards our products, and dig into expansion areas we’re considering.
+
+If you’re interested in volunteering your time, we invite you to sign up to talk to us. Look at the [products and features](https://calendly.com/cc-product-design) we’re currently scheduling tests for. If you’d like time to share general thoughts, sign up for a User Interview session specifically.
+
+## What to Expect
+- After scheduling a time to talk to us, here’s what you can expect:
+- You’ll need to sign and return a [consent form](https://drive.google.com/file/d/1XWt5QDu-IR76BwsPP7OidqNncha3rHbm/view) , allowing us to record the screen during the session. We typically do screen share, and the value of the session is greatly increased when we can refer back to a recording while reviewing notes and findings.
```

Comment:
There's an extra space after the "consent form" text before the comma, which should be removed.